### ISSUE TITLE
Can't resolve './createReducer'

### ISSUE THREAD

**Issue by anthonykrivonos** [frontendphil/react-undo-redo#28]
Hey, thanks for creating this package.

I added this via `yarn add react-undo-redo` and set it up according to the instructions in the README. When I run my app, however, I'm getting the following error:

```
./node_modules/react-undo-redo/lib/createContext.js
Module not found: Can't resolve './createReducer' in '.../node_modules/react-undo-redo/lib'
```

Any suggestions on how to fix it?

As a side note, I am not using Redux and I've tried `yarn add redux` and `yarn add @reduxjs/toolkit`.

**Comment by alvinleung** [comment]
I had the same problem.

What I did was download the repo and compile the module with yarn build manually and it worked! Seems like there is some problem with the version downloaded from npm. Hope it helps.

**Comment by keithpickering** [comment]
@alvinleung How were you able to import the library after manually building it? I tried this method but I'm getting the error `No present state found. Did you wrap your app in an UndoRedoProvider?` even though it is definitely wrapped as shown in the docs. Maybe I'm not doing it correctly.

**Comment by alvinleung** [comment]
@keithpickering Interesting, I didn't have that problem in my repo. How did you do the wrapping?

**Comment by alvinleung** [comment]
But anyways, I moved on to [use-undo](https://www.npmjs.com/package/use-undo) as I find this more stable. Though I still find myself liking api of react-undo-redo more :)

**Comment by frontendphil** [comment]
GitHub notifications are really great. I hadn't even noticed that people have trouble with the package. I'll try looking into this ASAP. From what I read it really seems like the published package is flawed.

**Comment by frontendphil** [comment]
@anthonykrivonos better late than never... 

I've just published `2.0.1` which should fix this. It was an issue with the build setup that went unnoticed because of how the demo page was built. I've changed my setup so that the demo page will always use the real deal which should highlight these issues faster.
